vaccine_feed_ingest/runners/ky/govstatus/normalize.py

```diff
@@ -0,0 +1,153 @@
+
+import datetime
+import json
+import logging
+import pathlib
+import sys
+
+from vaccine_feed_ingest_schema import schema  # noqa: E402
+
+logger = logging.getLogger(__name__)
+
+_source_name = "ky:govstatus"
+
+
+def _get_id(site: dict) -> str:
+    return site["id"]
+
+
+def _get_name(site: dict) -> str:
+    return site["name"]
+
+
+def _get_city(site: dict) -> str:
+    return site["address"]["city"]
+
+
+def _get_address(site: dict):
+    return schema.Address(
+        street1=site["address"]["street1"],
+        city=_get_city(site),
+        zip=site["address"]["zip"],
+        state=site["address"]["state"],
+    )
+
+
+def _get_location(site: dict):
+    latitude = site["lat"]
+    longitude = site["long"]
+    if latitude == "" or longitude == "":
+        return None
+    return schema.LatLng(
+        latitude=float(latitude),
+        longitude=float(longitude),
+    )
+
+
+def _get_contacts(site: dict):
+    ret = []
+
+    if "register_phone" in site:
+        raw_phone = site["register_phone"]
+        if raw_phone != "":
+            raw_phone = raw_phone.lstrip("tel:")
+            raw_phone = raw_phone.lstrip(" ")
+            raw_phone = raw_phone.lstrip("1")
+            raw_phone = raw_phone.lstrip("-")
+            raw_phone = raw_phone.lstrip(" ")
+            if raw_phone[3] == "-" or raw_phone[7] == "-":
+                phone = (
+                    "(" + raw_phone[0:3] + ") " + raw_phone[4:7] + "-" + raw_phone[8:12]
+                )
+                phone_notes = raw_phone[12:]
+            elif len(raw_phone) == 10:
+                phone = (
+                    "(" + raw_phone[0:3] + ") " + raw_phone[3:6] + "-" + raw_phone[6:10]
+                )
+                phone_notes = ""
+            else:
+                phone = raw_phone[0:14]
+                phone_notes = raw_phone[14:]
+
+            if phone_notes == "":
+                ret.append(schema.Contact(phone=phone, contact_type="booking"))
+            else:
+                phone_notes = phone_notes.lstrip(",")
+                phone_notes = phone_notes.lstrip(";")
+                phone_notes = phone_notes.lstrip(" ")
+                ret.append(
+                    schema.Contact(
+                        phone=phone,
+                        other=f"phone_notes:{phone_notes}",
+                        contact_type="booking",
+                    )
+                )
+
+    if "register_online_url" in site:
+        website = site["register_online_url"]
+        if website != "":
+            ret.append(schema.Contact(website=website, scontact_type="booking"))
+    return ret
+
+
+def _get_organization(site: dict):
+    # organization_name = ""
+    #
+    # if site["Organization Name"] == "":
+    #     return None
+
+    if _get_name(site)[0:6] == "Kroger":
+        return schema.Organization(name="Kroger", id="kroger")
+    if _get_name(site)[0:9] == "Walgreens":
+        return schema.Organization(name="Walgreens", id="walmart")
```

Comment:
Looks like a bad copy and paste. The id should be `walgreens` here

Reply:
Oops, fixed.